rotkehlchen/errors.py

```diff
@@ -90,10 +93,19 @@ class DeserializationError(Exception):
 
 
 class NoPriceForGivenTimestamp(Exception):
-    def __init__(self, from_asset: 'Asset', to_asset: 'Asset', date: str) -> None:
+    def __init__(self, from_asset: 'Asset', to_asset: 'Asset', date: 'Timestamp') -> None:
```

Comment:
Please also rename the timestamp argument.
```suggestion
    def __init__(self, from_asset: 'Asset', to_asset: 'Asset', time: 'Timestamp') -> None:
```